assets/dbs/interactable_objects_db.json

```diff
@@ -140,5 +140,28 @@
             "image": "assets/images/interactable_objects/psynergy_growth.png",
             "json": "assets/images/interactable_objects/psynergy_growth.json"
         }
+    },
+    {
+        "key_name": "move_hand",
```

Comment:
I think that this info will fit better in the file you created: `misc_animations_db.json`.